packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -105,15 +106,48 @@ export default class SaytDriverPlugin implements Plugin {
    * @param response An array search term strings.
    * @returns An array of search term strings.
    */
-  autocompleteCallback(response: AutocompleteResponse): string[] {
-    return response.result.searchTerms.map((term) => term.value)
-      .filter((term) => term);
+  autocompleteCallback(response: AutocompleteResponse): AutocompleteResponseSection[] {
+    const searchTerms = {
+      title: '',
+      items: response.result.searchTerms
+        ? this.constructSearchTerms(response.result.searchTerms)
+        : [],
+    };
+    return [searchTerms];
+  }
+
+  /**
+   * Formats a given list of search terms.
+   *
+   * @param terms An array of search terms.
+   * @returns An array of search terms that have been formatted.
+   */
+  constructSearchTerms(terms: AutocompleteSearchTerm[]): SearchTermItem[] {
+    return terms.filter((term) => term.value)
+      .map((term) => {
+        return { label: term.value };
+    });
   }
 }
 
 /**
  * The type of the sayt autocomplete request event payload.
  */
-export interface QueryTimeAutocompleteConfigWithQuery extends QueryTimeAutocompleteConfig {
+export interface AutocompleteConfigWithQuery extends QueryTimeAutocompleteConfig {
```

Comment:
I think `AutocompleteConfig` is sufficient here, unless we have one without `query`. Having `WithQuery` communicates to me that we have one that doesn't.